packages/@aws-cdk/aws-pipes-alpha/test/pipe.test.ts

```diff
@@ -186,6 +187,36 @@ describe('Pipe', () => {
       );
 
     });
+
+    test('grantDlqPush is called for sources with an SNS topic DLQ', () => {
+      // WHEN
+      const topic = new Topic(stack, 'MyTopic');
+      const sourceWithDlq = new TestSourceWithDlq(topic);
+
+      new Pipe(stack, 'TestPipe', {
+        pipeName: 'TestPipe',
+        source: sourceWithDlq,
+        target,
+      });
+
+      // THEN
+      expect(sourceWithDlq.grantDlqPush).toHaveBeenCalled();
```

Comment:
Fine to check for this but these tests should also check that the right grant has been added to the right role using our template matchers in our assert library.

Reply:
Done.